packages/es-components/src/components/controls/radio-buttons/RadioButton.js

```diff
@@ -120,4 +120,13 @@ RadioButton.propTypes = {
   theme: PropTypes.object
 };
 
+RadioButton.defaultProps = {
+  isChecked: false,
+  id: null,
+  isDisabled: false,
+  inline: false,
+  hasError: false,
+  theme: { colors: {}, sizes: {} }
```

Comment:
So I just realized that these defaults are effectively duplicated between here and the destructuring defaults [above](https://github.com/WTW-IM/es-components/blob/d064ca73e9fc1dceeaa634678455fec8cb54189d/packages/es-components/src/components/controls/radio-buttons/RadioButton.js#L64-L68). I wonder how much of this duplication is around with all these `defaultProps` additions.

Reply:
A lot, I'm sure. If you're curious about the reasoning around that, the [eslint-plugin-react rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/require-default-props.md) explains why they still expect `defaultProps` rather than default parameters. @Darrken and I talked about that during development.

From that explanation:
> One advantage of defaultProps over custom default logic in your code is that defaultProps are resolved by React before the PropTypes typechecking happens, so typechecking will also apply to your defaultProps. The same also holds true for stateless functional components: default function parameters do not behave the same as defaultProps and thus using defaultProps is still preferred.

I'm a little concerned about the potential for introducing bugs if I go through and try to remove all the possible duplication from that. Maybe it's something we could resolve to remove as we work in those areas.

Reply:
May as well remove the default parameters now in favor of `defaultProps` now. I don't see a whole lot of risk. If the tests break we can keep them around until we figure out how to remove them until they don't break. If the component has issues after removing the default params while the tests are passing I'm pretty sure we have other issues.